python/target_selection/cartons/mwm_yso.py

```diff
@@ -0,0 +1,576 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# @Author: Pramod Gupta (psgupta@uw.edu)
+# @Date: 2020-06-10
+# @Filename: mwm_yso.py
+# @License: BSD 3-clause (http://www.opensource.org/licenses/BSD-3-Clause)
+import peewee
+
+from sdssdb.peewee.sdss5db.catalogdb import (MIPSGAL, AllWise, Catalog,
+                                             CatalogToTIC_v8, Gaia_DR2, TIC_v8,
+                                             TwoMassPSC, YSO_Clustering)
+
+# from . import BaseCarton
+from target_selection.cartons import BaseCarton
+
+
+# See catalog.py for the name of peewee model names corresponding
+# to postgres table names:
+# https://github.com/sdss/sdssdb/blob/master/python/sdssdb/peewee/sdss5db/catalogdb.py
+
+
+class MWM_YSO_S1_Carton(BaseCarton):
+    """2.5.1. YSOs - S1 (IR excess)
+    Shorthand name: mwm_yso_s1
+Simplified Description of selection criteria:
+selection of YSOs based on IR excess,
+with WISE colors W1-W2>0.25, W2-W3>0.5, W3-W4>1.5,
+closer than parallax>0.3, and brighter than H<13
+(should have ~21.5K sources)
+Wiki page: https://wiki.sdss.org/display/MWM/YSO+selection+function
+Additional source catalogs needed: Gaia, 2mass, allwise
+Additional cross-matching needed:
+Note: Using the Gaia xmatch somehow misses half the sources.
+Selection was done on the allwise catalog that had 2mass photometry,
+and then the resulting selection was crossmatched against against
+Gaia with 1" search radius.
+Return columns: Gaia id, 2mass id, allwise id, G, BP, RP,
+J, H, K, W1, W2, W3, W4,parallax
+cadence options for these targets
+(list all options,
+even though no single target will receive more than one):
+Pseudo SQL (optional):
+Implementation: h_m<13 and w1mpro-w2mpro>0.25 and
+ w2mpro-w3mpro>0.5 and w3mpro-w4mpro>1.5 and parallax>0.3
```

Comment:
Could you make sure all the docstrings are indented, e.g.:

```python
class Test:
    """First line.
    
    Second line.
    Third line.

    """

    <code here>
```

The limit for docstring lines should be 72 chars where possible.